web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -159,14 +159,18 @@ internal fun Route.behandlingRoutes(
     post("$behandlingPath/{behandlingId}/tilAttestering") {
         // TODO: Short circuit
         call.withBehandling(behandlingService) { behandling ->
+            val navIdent = call.suUserContext.getNAVIdent()
+                ?: return@withBehandling call.svar(
```

Comment:
Kunne disse tilfellene bare gå på tryne sånn generelt? Slik at vi alltid kan deale med en "safe" navident i routsa?

Reply:
Egentlig så heller jeg mer og mer mot det ja, ref kommentaren min over her. Det vil føre til at vi _kanskje_ får noen unødige feil dersom Microsoft-API-et krøller (eller vår integrasjon mot det krøller), men, som du er inne på, det vil forenkle mye av koden vår. Samtidig kan vi også argumentere for at vi ikke burde tillate noen API-kall dersom vi ikke vet `navIdent` til brukeren som gjør kallet.